web/src/pages/index.page.tsx

```diff
@@ -50,8 +54,30 @@ const Home: NextPage = () => {
 
       <section id="how-it-works">
         <StyledBox>
-          <Typography variant="h4">How it works</Typography>
-          <YoutubeEmbed embedId="yM8RrwQWeJc" />
+          <Box sx={{ height: 600, width: "70%" }}>
```

Comment:
Do we need this div? Seems to me that any styling here can be put directly on the Carousel (by using emotion to make a `StyledCarousel`, like [this](https://github.com/amelioro/ameliorate/blob/1042b9615a55f69e02cc3a7d8805a977c2ad00a8/web/src/pages/index.style.tsx#L4-L10))

Reply:
Regarding the width - 70% is really small for smaller screens, but I do like having some margin. Maybe 90%?

Reply:
It's hard to remember, but this may have been an attempt to lock in a consistent height for all images- I think it was just experimental. Good point on using a `StyledCarousel` for formatting.

Reply:
generally `styled` is preferred over `sx` and `style` because
- it doesn't muddle up or mix with the html
- we can give meaningful names to the styled component (though we don't always do this)
- it applies css via class and therefore
  - allows control over related-element (e.g. child, sibling) css
  - allows overriding css
  - makes it easy to reuse css

`sx`/`style` can be fine if there are only a few, obviously one-off rules to apply. I'll mainly use them during prototyping if I haven't made a `styled` component yet, and only a few rules end up being used, I might just leave it partially out of laziness.